experiment/measurer/coverage_over_time_utils.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility DetailedCoverageData class and functions for report generation on
+segment and function coverage for the entire experiment
+(all Fuzzer-benchmark-trial combinations)."""
+
+import os
+import hashlib
+
+import pandas as pd
+
+from common import experiment_path as exp_path
+from common import filestore_utils
+from experiment.measurer import coverage_utils
+
+
+class DetailedCoverageData:  # pylint: disable=too-many-instance-attributes
+    """Maintains segment and function coverage information, and writes this
+    information to CSV files."""
+
+    def __init__(self):
+        """Constructor"""
+        self.segment_entries = []
+        self.function_entries = []
+        self.fuzzer_names = {}
+        self.benchmark_names = {}
+        self.function_names = {}
+        self.file_names = {}
+
+        # Will be initialized upon
```

Comment:
Change comment to: "Set by generate_data_frames_after_adding_all_entries()" please.

Reply:
Changed it.